x-pack/plugins/infra/public/components/asset_details/tabs/overview/metrics/metrics_grid.tsx

```diff
@@ -0,0 +1,286 @@
+    },
+    dataViewType: 'metrics',
+  },
+  {
+    id: 'diskThroughputReadWrite',
+    title: i18n.translate('xpack.infra.assetDetails.metricsCharts.diskIOPS', {
+      defaultMessage: 'Disk IOPS',
+    }),
+    layers: [
+      {
+        data: [
+          {
+            ...hostLensFormulas.diskReadThroughput,
+            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.metric.label.read', {
+              defaultMessage: 'Read',
+            }),
+          },
+          {
+            ...hostLensFormulas.diskWriteThroughput,
+            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.metric.label.write', {
+              defaultMessage: 'Write',
+            }),
+          },
+        ],
+        layerType: 'data',
+        options: {
+          seriesType: 'area',
+        },
+      },
+    ],
+    overrides: {
+      settings: LEGEND_SETTINGS.settings,
+    },
+    dataViewType: 'metrics',
+  },
+  {
+    id: 'diskIOReadWrite',
+    title: i18n.translate('xpack.infra.assetDetails.metricsCharts.diskThroughput', {
+      defaultMessage: 'Disk Throughput',
+    }),
+    layers: [
+      {
+        data: [
+          {
+            ...hostLensFormulas.diskIORead,
+            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.metric.label.read', {
+              defaultMessage: 'Read',
+            }),
+          },
+          {
+            ...hostLensFormulas.diskIOWrite,
+            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.metric.label.write', {
+              defaultMessage: 'Write',
+            }),
+          },
+        ],
+        layerType: 'data',
+        options: {
+          seriesType: 'area',
+        },
+      },
+    ],
+    overrides: {
+      settings: LEGEND_SETTINGS.settings,
+    },
+    dataViewType: 'metrics',
+  },
+  {
+    id: 'rxTx',
+    title: i18n.translate('xpack.infra.assetDetails.metricsCharts.network', {
+      defaultMessage: 'Network',
+    }),
+    layers: [
+      {
+        data: [
+          {
+            ...hostLensFormulas.rx,
+            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.network.label.rx', {
+              defaultMessage: 'Inbound (RX)',
+            }),
+          },
+          {
+            ...hostLensFormulas.tx,
+            label: i18n.translate('xpack.infra.assetDetails.metricsCharts.network.label.tx', {
+              defaultMessage: 'Outbound (TX)',
+            }),
+          },
+        ],
+        layerType: 'data',
+        options: {
+          seriesType: 'area',
+        },
+      },
+    ],
+    overrides: {
+      settings: LEGEND_SETTINGS.settings,
+    },
+    dataViewType: 'metrics',
+  },
+];
+
+export interface MetricsGridProps {
+  nodeName: string;
+  timeRange: TimeRange;
+  metricsDataView?: DataView;
+  logsDataView?: DataView;
+}
+
+export const MetricsGrid = React.memo(
+  ({ nodeName, metricsDataView, logsDataView, timeRange }: MetricsGridProps) => {
+    return (
```

Comment:
Should we show an error message in case `metricsDataView` and `logsDataView` are undefined? They must be passed in order for the component to work

Reply:
It's a good idea to have some indicators for similar cases also in other places ( Or to just not render some sections so it can be considered optional usage when they are not provided by the consumer/or have a warning that something is missing and show an error if we are not able to get them because of an error 🤔 )

Reply:
True. All other tabs have real optional parameters that we can pass in the `overrides` prop. That's not the case for the overview, these data views are mandatory. 
Not sure how we can best handle this. I'm inclined to throw an error, but I like your idea of not rendering the sections that depend on the data views and having a warning somewhere - perhaps in the console. Wdyt?

Reply:
Let's address this in a separate ticket.